content/influxdb/v2.0/reference/release-notes/flux.md

```diff
@@ -8,6 +8,13 @@ menu:
     name: Flux
 ---
 
+## v0.89.0 [2020-10-12]
+### Features
+- Add support for SAP HANA databases.
+- Add support for comments to builtin statements.
```

Comment:
...[comments](/influxdb/v2.0/reference/flux/language/lexical-elements/#comments) in built-in functions.

@nathanielc, is it still true that comments cannot start inside string or regexp literals?